meumobi/infomobi
# Feat/categories list

# Checklist

  - I've ran lint locally prior to submission.
- [x] I successfully ran tests locally with my changes.
  - Keep mockup up-to-date for tests purpose.
- [x] I've followed the guidelines from Contributing document (indicate which docs updated).
- NA I've updated wiki doc when necessary (indicate which docs updated).
- NA All actions are tracked on analytics (indicate which new actions).
- NA All expressions are translated.
- [x] All commits Close related issue, using `Closes #ISSUE_ID`.

# Tests

- login on existing site
- list categories on sidemenu, click on it and confirm items list from category selected are displayed
- click on category link from items list on home
